### DISCUSSION TITLE
[미니게임 강좌] 총알피하기 게임만들기

### DISCUSSION THREAD

**Discussion by Andwhy81kr** [naverz/zepeto-studio-kor#690] [3. Creator Samples]
제페토스크립트를 이용한 총알피하기 게임만들기 영상입니다.

[YouTube 영상링크](https://youtube.com/playlist?list=PLxGeHqAl0hZN7ZJh5SJmth9oa_18jGv2l)

제페토 스크립트 개발에 대해서 카페도 만들었으니 한번 들려서 개발정보 남겨주세요.

https://cafe.naver.com/zepetostudio

**Comment by smilejsu82** [comment]
멋져요 ~ ! 👍

**Comment by cwcc-cjlim** [comment]
와 감사합니다
